build/templates/errors.rst.mako

```diff
@@ -0,0 +1,63 @@
+<%
+    import build.helper as helper
+
+    config = template_parameters['metadata'].config
+    module_name = config['module_name']
+    driver_name = config['driver_name']
+    enums = config['enums']
+%>\
+${helper.get_rst_header_snippet('Exceptions and Warnings', '=')}
+
+${helper.get_rst_header_snippet('DriverError', '-')}
+
+    .. py:currentmodule:: ${module_name}
+
+    .. exception:: DriverError
+
+        An error originating from the ${driver_name} driver
+
+
+${helper.get_rst_header_snippet('UnsupportedConfigurationError', '-')}
+
+    .. py:currentmodule:: ${module_name}
+
+    .. exception:: UnsupportedConfigurationError
+
+        An error due to using this module in an usupported platform.
+
+${helper.get_rst_header_snippet('DriverNotInstalledError', '-')}
+
+    .. py:currentmodule:: ${module_name}
+
+    .. exception:: DriverNotInstalledError
+
+        An error due to using this module without the driver runtime installed.
+
+${helper.get_rst_header_snippet('InvalidRepeatedCapabilityError', '-')}
+
+    .. py:currentmodule:: ${module_name}
+
+    .. exception:: InvalidRepeatedCapabilityError
+
+        An error due to an invalid character in a repeated capability
```

Comment:
This is pretty specific, you could be more generic

`An error due to an invalid repeated capability`.

But then we're paraphrasing the class name.. so not sure.